### PR TITLE
fix: image common component with undefined src

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- When the image `src` prop is undefined, it doesn't throw an error which then sets the `fallbackSrc`.
- Fix the ad card fallback image to use our default for cards.

Preview before and after:

Screenshot 2024-02-16 at 6 04 28 AM

Screenshot 2024-02-16 at 6 04 21 AM

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-138 #done

### CODE REVIEW

File: packages/shared/src/components/cards/AdCard.tsx [hunk 1]
```diff
@@ -46,12 +47,14 @@ export const AdCard = forwardRef(function AdCard(
               showBlurredImage && 'absolute inset-0 m-auto',
             )}
             style={{ objectFit: showBlurredImage ? 'contain' : 'cover' }}
+            fallbackSrc={cloudinary.post.imageCoverPlaceholder}
```
[the remark this answers is not in the document]
Comment (sshanzel) [reply packages/shared/src/components/cards/AdCard.tsx:50]: Yeah, I mean this could've happened in the past, no way this doesn't have it 😂

**Comment by sshanzel** [comment]
> Maybe we could also consider setting the fallback in the Image component as a default?

Hmm. That can work, only concern is that it is used all across the app for any image implementation. If we can agree with that, then I think it should be good.

**Comment by rebelchris** [comment]
@sshanzel I think most usecases can do with fallback, it might not be the "right fit" but we rather solve those then not having fallback?
Let me know what you think.

**Comment by sshanzel** [comment]
At the moment, we do have a set of fallbacks depending on the usage. Maybe, we can add a prop (named `type`?) that would determine which fallback image to use, and if there is none (backwards support), we will utilize the card placeholder. Wdyt?

**Comment by rebelchris** [comment]
@sshanzel Sounds like a good plan indeed!

**Review by nensidosari — APPROVED** [review]

**Pull request merged**